sagemaker-inference-recommender/tensorflow-cloudwatch/tf-cloudwatch-inference-recommender.ipynb

```diff
@@ -0,0 +1,1439 @@
+{
```

Comment:
<p>Since notebook outputs should be cleared, consider including a JPG image of these example graphs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3088/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
<p>added</p>

Reply:
<p>Thanks for adding the image - I see it is hosted in another location outside of the repo (<code>https://aws-ml-blog-artifacts.s3.us-east-2.amazonaws.com/example-cw.png</code>). Can you add a copy of the image directly in the PR in case the original gets moved or taken down?</p>